modules/module_annex_change_history.adoc

```diff
@@ -7,40 +7,77 @@
 === General
 
 This annex documents the history of technical modifications made to
-ISO/TS 10303-1051.
+ISO/TS {docnumber}-{partnumber}.
 
-[yaml2text,changes_paths.yaml,data]
+
+[yaml2text,changes.yaml,context]
 ------
-{% assign max_version = 10 %} // This should be the {edition}
+
+{% liquid
+assign schema       = context.schema
+assign path_prefix  = "../../schemas/modules/"
+assign mapping_path = schema | prepend: path_prefix | append: "/mapping.changes.yaml"
+assign arm_path     = schema | prepend: path_prefix | append: "/arm.changes.yaml"
+assign mim_path     = schema | prepend: path_prefix | append: "/mim.changes.yaml"
+%}
+
+// This should be the {edition} but doesn't work inside yaml2text
+{% assign max_version = 10 %}
```

Comment:
@manuelfuenmayor can we use the value from `changes.yaml`?

https://github.com/metanorma/iso-10303-srl/blob/32b0046b402672bec34e8bdf41d34d57fe0867ab/documents/iso-10303-1003/changes.yaml#L4

Reply:
@ronaldtse `changes.yaml` does not contain the last version of change annex. No worries, I've solved it in https://github.com/metanorma/iso-10303-templates/pull/15/commits/f72618f2fa02bb4f8b3ccfcc3976ff062b28a6af.